scripts/loadDeployments.js

```diff
@@ -23,15 +23,15 @@ async function loadDeployment(network) {
   let contractsDeployed = {'contracts': {}};
   if (fs.existsSync('src/config/contracts.json'))
     contractsDeployed = JSON.parse(fs.readFileSync('src/config/contracts.json', 'utf-8'));
-
   const datContract = DATContract.at(proxies['BC-DAPP/DecentralizedAutonomousTrust'][0].address);
-
+  const fromBlock = (network == 'mainnet') ? 10000000 : (network == 'kovan') ? 18000000 : 0;
```

Comment:
why starting fromBlock from 10000000 and not 
10012634 which was the block that deployed the contract?

https://etherscan.io/tx/0x5ba97e7ceb6b409267e8a07d33dbc0a814ef5ea3eae9e2077ecea2b9ccc60a6d